src/components/form/range/range.test.tsx

```diff
@@ -7,7 +7,7 @@
  */
 
 import React from 'react';
-import { fireEvent } from '@testing-library/react';
+import { fireEvent, getByRole } from '@testing-library/react';
```

Comment:
We should be getting `getByRole` from the `render()` util, not from the global library
```suggestion
import { fireEvent } from '@testing-library/react';
```